cpp/src/backends/torch_scripted/torch_scripted_backend.cc

```diff
@@ -74,14 +74,51 @@ std::unique_ptr<torchserve::LoadModelResponse> Backend::LoadModelInternal(
 
 std::shared_ptr<torchserve::InferenceResponseBatch> ModelInstance::Predict(
     std::shared_ptr<torchserve::InferenceRequestBatch> request_batch) {
+  auto predict_start_time = std::chrono::high_resolution_clock::now();
+  std::string request_id_batch = "";
   auto response_batch = std::make_shared<torchserve::InferenceResponseBatch>();
   for (const auto& request : (*request_batch)) {
     (*response_batch)[request.request_id] =
         std::make_shared<torchserve::InferenceResponse>(request.request_id);
+    request_id_batch += request_id_batch.empty() ? request.request_id
+                                                 : "," + request.request_id;
```

Comment:
idx_to_req_id.first build the string. Check [PR](https://github.com/pytorch/serve/blob/69a372290b6381630564fe5e49cc4de0a894e955/cpp/src/backends/torch_scripted/handler/base_handler.cc#L81)

Reply:
Updated implementation